tilavarauspalvelu/models/reservation/actions.py

```diff
@@ -43,14 +43,57 @@ def get_actual_after_buffer(self) -> datetime.timedelta:
         return buffer_time_after
 
     def to_ical(self) -> bytes:
-        site_name = settings.EMAIL_VARAAMO_EXT_LINK
         language: Lang = (  # type: ignore[assignment]
             self.reservation.reservee_language
             or (self.reservation.user is not None and self.reservation.user.get_preferred_language())
             or settings.LANGUAGE_CODE
         )
 
+        cal = Calendar()
+        cal.add(CalendarProperty.VERSION, "2.0")
+        cal.add(CalendarProperty.PRODID, "-//Helsinki City//NONSGML Varaamo//FI")
+
+        cal.add_component(self._get_ical_timezone())
+        cal.add_component(self._get_ical_event(language=language))
+
+        return cal.to_ical()
+
+    def _get_ical_timezone(self) -> Timezone:
+        """Adds timezone information to the ical calendar event."""
+        timezone = Timezone()
+        timezone.add(TimezoneProperty.TZID, settings.TIME_ZONE)
+
+        # Taken from outlook generated iCal files
```

Comment:
Could extend the comment that these assumptions are valid for timezone which observe European Summer Time as currently defined (EU directive 2000/84/EC).
If in the future EU eventually implements the proposal to abolish DST, then an UNTIL definition needs to be added to the RRULEs (so that the icals will still be generated properly across the transition period).

Reply:
Added clarifying comments to timezone use.